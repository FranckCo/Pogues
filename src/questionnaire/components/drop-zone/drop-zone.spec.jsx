import React from 'react';
import renderer from 'react-test-renderer';
import DropZone from './drop-zone';

describe('DropZone', () => {
  test('Should have this default template', () => {
    const style = {};
    const tree = renderer.create(<DropZone style={style} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
