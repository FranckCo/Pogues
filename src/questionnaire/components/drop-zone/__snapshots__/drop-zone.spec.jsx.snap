// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropZone Should have this default template 1`] = `
<div
  className="questionnaire-element-drop-zone"
  style={Object {}}
>
  <div
    className="questionnaire-element-label"
  >
    Drop here !
  </div>
</div>
`;
